tests/erp/test_contrib.py

```diff
@@ -122,7 +122,7 @@ def test_contrib_start_global_search_with_existing(
         return_value=[mairie_jacou_result, akei_result],
     )
 
-    obj_erp = ErpFactory(
+    _obj_erp = ErpFactory(
```

Comment:
if obj_erp is not used, simply do not assign the factory to a var.

```suggestion
    ErpFactory(
```